CliWrap.Tests/CliTests.cs

```diff
@@ -131,6 +133,40 @@ public void Execute_EchoSpam_Callback_Test()
             AssertExecutionResult(result, 0, expectedStandardOutput, expectedStandardError);
         }
 
+        [Test]
+        public void Execute_Sleep_InSubprocess_CancelEarly_Test()
+        {
+            // Arrange & act & assert
+            using (var cts = new CancellationTokenSource())
+            {
+                cts.Cancel();
+
+                Assert.Throws<OperationCanceledException>(() =>
+                {
+                    Cli.Wrap(SleepInSubprocessBat)
+                        .SetCancellationToken(cts.Token)
+                        .Execute();
+                });
+            }
+        }
+
+        [Test]
+        public void Execute_Sleep_InSubprocess_CancelLate_Test()
+        {
+            // Arrange & act & assert
+            using (var cts = new CancellationTokenSource())
+            {
+                cts.CancelAfter(TimeSpan.FromSeconds(1));
+
+                Assert.Throws<OperationCanceledException>(() =>
+                {
+                    Cli.Wrap(SleepInSubprocessBat)
+                        .SetCancellationToken(cts.Token)
+                        .Execute();
+                });
+            }
+        }
+
```

Comment:
These tests don't seem to be making sure that child processes are killed. In fact, `OperationCanceledException` will be thrown either way. It appears that these tests don't add value on top of the previously existing `Execute_Sleep_CancelEarly_Test` tests.